src/plugins/core/range.ts

```diff
@@ -384,6 +387,25 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
     return { _sheetId: sheetId, _zone: zone };
   }
 
+  getRangeFromZone(
+    sheetId: UID,
+    zone: UnboundedZone,
+    parts: RangePart[] = [
```

Comment:
`parts` is unused. You can remove the parameter

Reply:
It is used in the RangeImpl constructor right behind

Reply:
Oh, okay, I got what you mean